src/v3/src/util/idxUtils.ts

```diff
@@ -355,3 +357,18 @@ export const isVerifyFlow = (transaction: IdxTransaction): boolean => {
 
 // @ts-expect-error OKTA-627610 captcha missing from context type
 export const isCaptchaEnabled = (transaction: IdxTransaction): boolean => typeof transaction.context?.captcha !== 'undefined';
+
+export const isConsentStep = (transaction: IdxTransaction | undefined): boolean => (
```

Comment:
```suggestion
export const isConsentStep = (transaction?: IdxTransaction): boolean => (
```

same for `getApplicationName`